README.md

```diff
@@ -1 +1,152 @@
-# capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Please add the correct logo image path in your `README` file. Right now your logo link is broken because the image that you are using is not correct and doesn't exist. Kindly fix it either by removing this section or adding a valid logo image path.